index.js

```diff
@@ -1,60 +1,73 @@
-const fs = require("fs"),
```

Comment:
You should not touch this file

Reply:
I had to download the editorconfig npm and import it into this file